### DISCUSSION TITLE
posts/2018/sphinx-copy-buttons/

### DISCUSSION THREAD

**Discussion by giscus** [choldgraf/choldgraf.github.io#24] [Blog comments]
# posts/2018/sphinx-copy-buttons/

NOTE: This is now a sphinx extension! Thanks to some friendly suggestions, I’ve written this up as a super tiny sphinx extension. Check it out here: https://github.com/choldgraf/sphinx-copybutton S...

https://predictablynoisy.com/posts/2018/sphinx-copy-buttons/?highlight=custom+css

**Comment by gloc-mike** [comment]
Thanks for the handy sphinx extension! Brilliant! 🎉 ☺️

I was struggle with implementing another solution and your just work! 🍾
